kotsadm/pkg/redact/redact.go

```diff
@@ -186,6 +186,52 @@ func SetRedactYaml(slug, description string, enabled, newRedact bool, yamlBytes
 	return redactorEntry, nil
 }
 
+// sets whether an individual redactor is enabled
+func SetRedactEnabled(slug string, enabled bool) (*RedactorMetadata, error) {
+	configMap, _, err := getConfigmap()
+	if err != nil {
+		return nil, err
```

Comment:
errors.wrap?

Reply:
getConfigmap returns wrapped errors already: https://github.com/replicatedhq/kots/blob/cf3809d6209d06d12b2fa2a21db5e4fb2f5072cc/kotsadm/pkg/redact/redact.go#L327

Adding another layer doesn't seem useful here

Reply:
that being said, we're inconsistent in whether this function has errors wrapped
I'll just add it everywhere

Reply:
Okay, thanks